angular/projects/spark-angular/src/lib/components/sprk-ordered-list/sprk-list.stories.ts

```diff
@@ -0,0 +1,106 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkOrderedListModule } from './sprk-ordered-list.module';
+import { SprkUnorderedListModule } from '../sprk-unordered-list/sprk-unordered-list.module';
+import { SprkListItemModule } from '../sprk-list-item/sprk-list-item.module';
+
+export default {
+  title: 'Components|List',
+  component: SprkUnorderedListModule,
+  decorators: [
+    storyWrapper(
+      storyContent => (
+        `<div class="sprk-o-Box sprk-u-mlm">${ storyContent }<div>`
+      )
+    )
+  ],
+};
+
+const modules = {
+  imports: [
+    SprkOrderedListModule,
+    SprkUnorderedListModule,
+    SprkListItemModule,
+  ],
+};
+
+export const unordered = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-unordered-list idString="unordered-list-1">
+      <sprk-list-item idString="list-item-1">
+        Unordered List Item
+      </sprk-list-item>
+      <sprk-list-item idString="list-item-2">
+        Unordered List Item
+      </sprk-list-item>
+      <sprk-list-item idString="list-item-3">
+        Unordered List Item
+      </sprk-list-item>
+    </sprk-unordered-list>
+  `,
+});
+
+export const ordered = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-ordered-list idString="ordered-list-1">
+      <sprk-list-item idString="list-item-4">
+        Ordered List Item
+      </sprk-list-item>
+      <sprk-list-item idString="list-item-5">
+        Ordered List Item
+      </sprk-list-item>
+      <sprk-list-item idString="list-item-6">
+        Ordered List Item
+      </sprk-list-item>
+    </sprk-ordered-list>
+  `,
+});
+
+export const indented = () => ({
+  moduleMetadata: modules,
+  template: `
+    <p>
```

Comment:
I know we had these paragraphs in there before, but they aren't related to the component itself. They are kinda confusing as to why they are there to people, of course we know we put them there for illustration purposes of the indentation, but that doesn't make sense in the SB.